### PR TITLE
Add Readme, replace logo, update 'about' section

### PR DESCRIPTION
**Pull request by daracline**
Previously, the staff cards were giant and smooshed together. This updates them to be smaller and better spaced.

### CODE REVIEW

**Review by BastionFennell — APPROVED** [review]
Looks good to me! Go ahead and merge whenever you're ready

**Review by BastionFennell — COMMENTED** [review]
This is awesome! How much of a pain would it be to pull each issue we're fixing into separate PRs?

**Comment by daracline** [comment]
Oh, totally didn't see that message until just now lol. But I can definitely do that from now on

**Comment by daracline** [comment]
Should I leave it unmerged for now since I haven't actually fixed the button spacing? Or should I merge it anyway?

**Comment by BastionFennell** [comment]
Go ahead and add the button spacing, I have a couple small nitpicks just to clean things up

File: index.html
Comment (BastionFennell) [inline index.html]: Do we still need these lines?

File: stylesheet.css
Comment (BastionFennell) [inline stylesheet.css]: Is this line important?

File: stylesheet.css
Comment (BastionFennell) [inline stylesheet.css]: Nit: indention

File: stylesheet.css
Comment (BastionFennell) [inline stylesheet.css]: Can we remove this white space?

File: index.html
Comment (BastionFennell) [inline index.html]: Just to be consistent, it might make sense to kebab-case this class

File: index.html
Comment (BastionFennell) [inline index.html]: Is there something more descriptive we can use instead of `about-p1` and `about-p2`

**Review by BastionFennell — CHANGES_REQUESTED** [review]
Overall looks good! We did get a .DS_Store in here, maybe we should add that to our .gitignore file? Let's at least make sure we don't accidentally pull it into master :D

It might also make sense to update the title and description of the PR to better reflect the work we're pulling in

File: index.html
Comment (daracline) [reply index.html]: Nope. Cleaning these up as we speak. Although, I might run and get some food first lol

File: stylesheet.css
Comment (daracline) [reply stylesheet.css]: lol I'll fixt this, too.

**Comment by daracline** [comment]
I think I fixed all the nitpicks, but I'm too paranoid to merge it to the master lol. Does everything look good now?

**Comment by BastionFennell** [comment]
Good call, let's make sure we have approvals from one of the other devs before we merge to master :D

**Review by BastionFennell — APPROVED** [review]
LGTM!

File: index.html
Comment (BastionFennell) [inline index.html]: I'm not sure if putting this height and width here is doing anything

File: stylesheet.css
Comment (BastionFennell) [inline stylesheet.css]: Is this still important? It looks like we changed the class name for the image itself

File: index.html
Comment (daracline) [reply index.html]: Is this an earlier version? Because this doesn't match my most recent commit.

Comment (BastionFennell) [reply index.html]: Interesting, I'm not sure. It's what GitHub thinks you've added at least :P

**Pull request merged**
